File: Data Structures & Algorithms/max-consecutive-ones/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    findMaxConsecutiveOnes(nums) {
     let maxx=0
    let i=0
    let curr=0
    while(i<nums.length){
        if(nums[i]!==1){
           
            curr=0
        }else{
            curr+=1
        }
        maxx=Math.max(maxx,curr)
        i+=1
    }
    return maxx
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if len(nums)==0:return 0
        if len(nums)==1: return 1
        nums.sort()
        maxx=1
        currmax=1
        prev=nums[0]
        for i in range(1,len(nums)):
            if nums[i]==prev+1:
                maxx+=1
                currmax=max(maxx,currmax)
            elif nums[i]==prev:
                continue
            else: 
                currmax=max(maxx,currmax)
                maxx=1
            prev=nums[i]
        return currmax




        
